package com.exceptionHandling.self;

public class Mock {
	public static void main(String[] args) {
		
//		System.out.println("start");
//		new NullPointerException("hello vishnu").printStackTrace();
//	System.out.println(new NullPointerException("hello vishnu").getMessage());
//		System.out.println("end");
//		
		
		
		

//			System.out.println("Start ");
//
//			// Task6 t = new Task6()
//			// new Task6().task();
//			new NullPointerException("hello").printStackTrace();
//
//			System.out.println(new NullPointerException("hello").getMessage());
//			System.out.println(new NullPointerException("hello").toString());
//
//			
//		 	NullPointerException exp =new NullPointerException("Example");
//		 	System.out.println(exp);
//		 	
//			System.out.println("end");
//		}
//
//		public void task() {
//			System.out.println("welcome to task");
//			System.out.println(10 / 0); // new AE("/ by 0");
//			System.out.println("bye task");
//		}
		
		
		
		
		
		
		
}
}
